### DISCUSSION TITLE
Should we add "How to look for an open source project to contribute" on the support FAQ?

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#275] [Ideas] [answered]
**Original discussion**:
![imagem](https://user-images.githubusercontent.com/18630253/94738026-b6010600-0366-11eb-9974-8a56564ca004.png)

What is everyone's opinion on this? I think adding the list that @eddiejaoude said provides value to a beginner. Instead of losing this information on Discord chat, we can add it to the `README`.
I also found [this link](http://opensource.guide/how-to-contribute/) when searching this topic on Google 😃

**Comment by eddiejaoude** [comment]
Yes lets do it 👍 - I am sure it could be improved also.

**Comment by mikeysan** [comment, marked as answer]
Good call. I saw something similar on the Hactoberfest website that was massively outdated (I tweeted them about it and it looks like they've removed it now). It appears the maintainer had given up on the project. Unfortunately, It is/was above my skill level to clone it and improve upon it. Maybe it can give us a starting point (I'll look for it again).

update:
It took a while, but I found it. https://github.com/davearonson/pull-request-roulette

**Comment by BOLT04** [comment]
okay awesome 👍, not sure if we should create an issue or just a PR and point to this Discussion

**Reply by eddiejaoude** [reply]
We should probably add it to the README or its own `md` file that we can direct people too and be in the README. What do you all think?

**Reply by BOLT04** [reply]
yes I like that idea of it being in a different file 👍 👍. Then we could have a link to it on the FAQ of any repo, or even the website 😃.
For that reason, I think this file should be on the `support` repo, with the question "How to look for an open-source project to contribute" on the FAQ inside the `README` that links to that file. If in the future we create more FAQs on the other repos we add the same link.

**Reply by eddiejaoude** [reply]
Yep `support` repo is a great place to central our docs 👍

**Reply by eddiejaoude** [reply]
This could be created into an Issue on the `support` repo
